ctxhandler.go

```diff
@@ -184,48 +175,14 @@ func ContextWithGroup(ctx context.Context, group string) context.Context {
 // LaxCtxHandler is an option for disable adding !BADCTX attr.
 func LaxCtxHandler() ctxHandlerOption { //nolint:revive // By design.
 	return func(ctxHandler *CtxHandler) {
-		ctxHandler.ignoreBADCTX = true
+		ctxHandler.omitBadCtx = true
 	}
 }
 
-func (h *CtxHandler) clone() *CtxHandler {
+func (h *CtxHandler) addOp(op handlerOp) *CtxHandler {
```

Comment:
Maybe `withOp` is a better name and more inline with `WithGroup` and `WithAttrs`.
I'm not insisting on this, but "add" feels to me more like it'll change existing object instead of returning a new one.

And/or it might make sense to change receiver type to not a pointer, to make it clear this method won't modify existing object. Also in this case implementation may be modified to became a bit more robust: instead of defining all 3 fields (which will needs to be updated in case new field will be added) we can just change receiver's `ops` field.